.env.example

```diff
@@ -40,6 +40,7 @@ JWT_DONATION_SECRET=myLocalJwtSecret
 JWT_ID_SECRET=myLocalIdSecret
 
 SALESFORCE_CLIENT_TIMEOUT=15
+SALESFORCE_API_BASE=https://sf-api-staging.thebiggivetest.org.uk/
```

Comment:
IMHO this should replace the four environment variables below - but we can't delete them until after this new one is set in prod.